tendermint/src/rpc/endpoint/net_info.rs

```diff
@@ -104,83 +97,43 @@ pub struct Monitor {
     pub start: Time,
 
     /// Duration of this monitor
-    #[serde(
-        rename = "Duration",
-        serialize_with = "serializers::serialize_duration",
-        deserialize_with = "serializers::parse_duration"
-    )]
+    #[serde(rename = "Duration", with = "serializers::timeduration::string")]
```

Comment:
Do we anticipate other ways to encode/decode `std::time::Duration`? If not, the extra string sub-module isn't necessary I think (and this could be `serializers::timeduration`).

Reply:
I have no idea, that's why I just kept the above mentioned "serialize from A to B" type of notion. Again, adding more modules just helps with the `::` annotation, it doesn't serve any other purpose. I'd say, if we have a list of annotation names you'd like to see, I'll just rename everything to it.

Reply:
Not sure how I feel introducing a wrapping module just to keep the documentation consistent. Seems like an uncommon pattern to me (but I might be wrong).

Reply:
I mentioned renaming it together with the other types in a comment above. Let's continue there.